src/client/App.js

```diff
@@ -58,15 +59,19 @@ function App() {
         </NonAuthenticatedRoute>
 
         {/* All routes below are authenticated routes - a user must login first */}
+        <Route exact path="/order/:id">
+          <OrderPageContainer />
+        </Route>
+
         <AuthenticatedRoute exact path="/profile">
           <ProfilePage />
         </AuthenticatedRoute>
         {/* Make sure to keep wildcard "*" routes in the bottom of the Switch */}
         <Route path="*">
           <Page404Container />
         </Route>
+        <Footer />
       </Switch>
-      <Footer />
```

Comment:
I think you shouldn't make this changes. Because this changes will affect all container pages